README.md

```diff
@@ -3,6 +3,47 @@ A repository for developing the User Interface for receiving data and controllin
 
 
 ## Packages Info
+# Autolaunch
+
+This guide will show you how to set up and launch the MRCS (Manual Rover Control System) using ROS 2 on a Docker bash terminal.
+
+## Instructions
+
+1. Open your ROS 2 Docker terminal.
+2. Install `tmux` by running `sudo apt-get install tmux`.
+   - `tmux` is a terminal multiplexer that allows you to create multiple windows and panes within a single terminal window.
+3. Create a new directory named `ros2_ws` with a subdirectory called `src` using `mkdir -p ros2_ws/src`.
+4. Navigate to the subdirectory inside the folder you just created by running `cd ros2_ws/src`.
+5. Clone the latest version of the MRCS repo using `git clone git@github.com:GonzagaRobotics/ManualRoverControlSystem.git`.
+6. Change back into `ros2_ws` by running the command `cd ros2_ws`.
+7. Build the project by running `colcon build`.
+   - If you encounter the error message `stderr: teleop_twist_joy`, run `colcon build` again to resolve it.
+8. Source ROS2 in the parent directory by running the command: `. install/setup.bash`.
+9. Startup `tmux` by running the command `tmux`.
+   - If a green status bar appears at the bottom of your terminal, move on to the next step. Otherwise, try step 9 again, and if that doesn't work, try step 1 to make sure it is installed correctly on your system.
+10. Before moving on, note that all commands in `tmux` are triggered by a prefix key followed by a command key. For our use case, the "prefix key" is the keyboard shortcut `CTRL+B`.
+11. Press `CTRL+B` followed by a ` SHIFT+ ‘` or `"` to open a top and bottom split-screen terminal.
+12. Now that you have two terminal panels, use `CTRL+B` followed by the `UP` arrow to navigate to the top panel.
+13. In the top panel, run the launch file command by running `ros2 launch src/ManualRoverControlSystem/autolaunch/launch/auto_launch.py`. 
+You can stop this launch process at any time by pressing `CTRL+C`. The output should look something like this:
+    - [INFO] [launch]: All log files can be found below /root/.ros/log/2023-02-25-03-11-34-708275-2f922f3c3a13-1665
+    - [INFO] [launch]: Default logging verbosity is set to INFO
+    - [INFO] [joy_node-1]: process started with pid [1666]
+    - [INFO] [XboxBroker-2]: process started with pid [1668]
+    - [INFO] [CommandExposer-3]: process started with pid [1670]
```

Comment:
Good extra info, but put this in a multiline code block, which can be done in markdown with triple \`